src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -5,22 +5,30 @@ import { closestComposedNode, getOrInsertCSSRule } from '../utils/element-utils.
 import { MediaStateReceiverAttributes } from '../constants.js';
 
 const template = document.createElement('template');
-template.innerHTML = `
+template.innerHTML = /*html*/`
   <style>
   :host {
     display: inline-flex;
     position: relative;
     flex-shrink: .5;
+    background: var(--media-control-background, rgba(20,20,30, 0.7));
+    padding: var(--media-control-padding, 10px);
   }
 
   [name="listbox"]::slotted(*),
-  [part=listbox] {
+  [part="listbox"] {
     position: absolute;
     left: 0;
     bottom: 100%;
     max-height: 300px;
     overflow: hidden auto;
   }
+
+  [name="button"]::slotted(*),
+  [part="button"] {
+    padding: 0;
+    background-color: inherit;
```

Comment:
Could probably just make this transparent, yeah? Any kind of gradient could we weird/missed otherwise, but maybe that's too different from how selectmenus are supposed to work? Good either way.

Reply:
I think yes, will test this soon